src/WebBrowser-Core.package/WebBrowser.class/class/webbrowserClassForPlatform.st

```diff
@@ -0,0 +1,8 @@
+private - accessing
+webbrowserClassForPlatform
+
+	Smalltalk os isUnix ifTrue: [ ^WBUnixWebBrowser ].
+	Smalltalk os isWindows ifTrue: [ ^WBWindowsWebBrowser ].
+	(Smalltalk os isMacOS or: [ Smalltalk os isMacOSX ])  
+		ifTrue: [ ^WBMacWebBrowser ].
```

Comment:
The class name "WebBrowser" is already used in many external packages.

Also you would like to write   WebBrowser openOn: aURL
instead of AbstractWebBrowser openOn: aURL

Independent from that I added the extensibility mechanism by searching for appropriate subclass - although I think it will never gonna used soon ;)